sdk/search/Microsoft.Azure.Search.Service/src/Generated/Models/IndexerExecutionInfo.cs

```diff
@@ -37,11 +37,13 @@ public IndexerExecutionInfo()
         /// in-progress indexer execution.</param>
         /// <param name="executionHistory">History of the recent indexer
         /// executions, sorted in reverse chronological order.</param>
-        public IndexerExecutionInfo(IndexerStatus status = default(IndexerStatus), IndexerExecutionResult lastResult = default(IndexerExecutionResult), IList<IndexerExecutionResult> executionHistory = default(IList<IndexerExecutionResult>))
+        /// <param name="limits">The execution limits for the indexer.</param>
+        public IndexerExecutionInfo(IndexerStatus status = default(IndexerStatus), IndexerExecutionResult lastResult = default(IndexerExecutionResult), IList<IndexerExecutionResult> executionHistory = default(IList<IndexerExecutionResult>), IndexerLimits limits = default(IndexerLimits))
```

Comment:
This is a binary breaking change, assuming you have already shipped the older api. If you haven't shipped it then this is fine but if you have shipped it then you need to add an overload to ensure there isn't a breaking change.

Reply:
@weshaggard - We've considered making this breaking change, which is why @enjaimes has bumped up the major version and make a release later in the month of August.
Please do merge this PR in whenever possible.